### ISSUE TITLE
EntityList table headers don't align properly on version 7.0 beta-5

### ISSUE THREAD

**Issue by uwascan** [code16/sharp#369]
![Screenshot_1](https://user-images.githubusercontent.com/5307578/146283345-834411fa-b1bd-41ec-926f-dcde18744004.png)

**Comment by dvlpp** [comment]
Ouch! Could you please share your `buildListLayout()` code?

**Comment by uwascan** [comment]
```
    protected function buildListLayout(EntityListFieldsLayout $fieldsLayout): void
    {
        $fieldsLayout->addColumn("name")
            ->addColumn("contact_name", 2)
            ->addColumn("contact_phone", 2)
            ->addColumn("contact_email", 3);
    }
```

**Comment by uwascan** [comment]
I just discovered that the EntityList table headers align properly only when you have Instance Commands as shown below
![Screenshot_1](https://user-images.githubusercontent.com/5307578/146382355-ad169041-66dc-4cee-9fe5-43b3349985f4.png)

**Comment by uwascan** [comment]
Quick question: Is possible to make Custom Show Fields have access to base Sharp View Components? This will enable one build custom ui that shares same look and feel with Sharp in terms of styling. I tried using ``` SharpText ``` in my Custom Show Field, it built but nothing was displayed
```<SharpText value="Single SHOW" :placeholder="'PUT SOME HERE'" :readOnly="false"></SharpText>```

**Comment by aguingand** [comment]
Fixed in v7.0.0-beta.6

**Comment by aguingand** [comment]
> Quick question: Is possible to make Custom Show Fields have access to base Sharp View Components?

Its currently not possible to access internal components as it may change at any time (prop names, etc). But as sharp 7 is using bootstrap for styling, you can use default bootstrap classes (form-control, etc).

If you have another question or want to continue this discussion, please create a [discussion](https://github.com/code16/sharp/discussions)
